tartufo/cli.py

```diff
@@ -146,7 +146,21 @@ def get_command(self, ctx: click.Context, cmd_name: str) -> Optional[click.Comma
     callback=config.read_pyproject_toml,
     help="Read configuration from specified file. [default: tartufo.toml]",
 )
-
+@click.option(
+    "-q/ ",
+    "--quiet/--no-quiet",
+    help="Quiet mode. No outputs are reported if the scan is successful and doesn't find any issues",
+    default=False,
+    is_flag=True,
+)
+@click.option(
+    "-v/ ",
+    "--verbose/--no-verbose",
+    help="Verbose mode. When this flag is provided, tartufo will report excluded paths and signatures"
+    " as well as scan issues",
```

Comment:
I think we may want this to be a `count`, rather than a flag, like we see here in the `click` documentation: https://click.palletsprojects.com/en/7.x/options/#counting

The idea here is that, once we switch to using a proper logger, we use this to change the logging level. So for example, by default we would probably be at `logging.ERROR`; with `-v` it would bump up to `logging.WARNING`; `-vv` would go to `logging.INFO`; and `-vvv` would go to `logging.DEBUG`.

The other implication here is that I think the help text should be more generic. Something along the lines of:

> Display more verbose output. Specifying this option multiple times will incrementally increase the amount of output.

Reply:
We should also make sure that `-q` and `-v` are mutually exclusive. Perhaps throw an error if both are specified.

Reply:
Added validation to catch this: 
```
if options.quiet and options.verbose > 0:
    raise click.BadParameter("-v/--verbose and -q/--quiet are mutually exclusive.")
```

Reply:
Updated `--verbose` to be a `count` with `default=0` and `int` in `GlobalOptions`. Updated help text.